quickstarts/completions/.eslintrc

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
should we add eslint as a dev dependency?

Reply:
done